src/main/java/com/softdevsix/argos/domain/ProjectRules.java

```diff
@@ -92,4 +92,8 @@ public Coverage getCoverage() {
   public Integer getId() {
     return id;
   }
+
```

Comment:
please dont define setters if it is truly necessary use @Value from lombok to create getters and @builder from lombok to construct the object